stackslib/src/net/chat.rs

```diff
@@ -1999,9 +1984,29 @@ impl ConversationP2P {
         ) {
             Ok(Some(chunk)) => chunk,
             Ok(None) => {
-                // request for a stale chunk
+                // TODO: this is racey
```

Comment:
If you want to merge this with TODOs present, they should at least link to open issues to ensure they are not forgotten.